app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectTypes.jsx

```diff
@@ -101,6 +102,10 @@ export const DefectTypes = ({ setHeaderTitleNode }) => {
     );
   };
 
+  const onDocumentation = () => {
+    trackEvent(PROJECT_SETTINGS_DEFECT_TYPES_EVENTS.CLICK_DOCUMENTATION_LINK);
```

Comment:
As a suggestion, we can check e.target and if e.target tagName = 'a' send event.
What do u think?

Reply:
Done.